test/module/Entra/Set-EntraAdministrativeUnit.Tests.ps1

```diff
@@ -0,0 +1,33 @@
+BeforeAll {  
```

Comment:
Add license information at the top of this file

Reply:
done